spec/services/spam/handler_spec.rb

```diff
@@ -104,6 +104,14 @@
       allow(Settings::General).to receive(:mascot_user_id).and_return(mascot_user.id)
     end
 
+    context "when using :more_rigorous_user_profile_spam_checking" do
+      before do
+        allow(FeatureFlag).to receive(:enabled?).with(:more_rigorous_user_profile_spam_checking).and_return(true)
+      end
+
+      it { is_expected.to eq(:not_spam) }
+    end
+
```

Comment:
Question: Would it be worth adding the compliment spec as well? 

What I mean is we're only checking that the `handler_user!` method is not marking these as spam, but there isn't a spec to cover the other scenario: Where a user has a spam keyword in their GitHub username, their website, etc ([like it is here](https://github.com/forem/forem/blob/167732dea60fe97f564ee023fecc7a873aa5ae2d/spec/services/spam/handler_spec.rb#L123-L131))

Reply:
I had considered that…but opted to not.  However, I can definitely add it in.